<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalesLogin</class>
 <widget class="QWidget" name="SalesLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SalesLogin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#SalesLogin{
	
	background-image: url(:/image/back.jpg);
}</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>40</y>
     <width>551</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文细黑</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>快跑校园订餐系统服务器</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>407</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文细黑</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>管理员登录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="le_pswd">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>admin</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>密码:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QPushButton" name="pb_register">
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>注册账号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户名:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="le_id">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>admin</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QPushButton" name="pb_login">
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pb_login{
    background-color: rgb(255, 85, 0);
    border-style: outset;
    border-width: 4px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
}
QPushButton#pb_login:pressed {
    background-color: rgb(237, 255, 245);
    border-style: inset;
}</string>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QPushButton" name="pb_forget">
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>忘记密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
